R/FilterStateExpr.R

```diff
@@ -106,7 +111,7 @@ FilterStateExpr <- R6::R6Class( # nolint
     #' @description
     #' Sets filtering state.
     #'
-    #' @param state a `teal_slice` object
+    #' @param state (`teal_slice`) object
```

Comment:
@kartikeyakirar same as in other places for `param` - I would remove `object`

Reply:
[95ced22](https://github.com/insightsengineering/teal.slice/pull/506/commits/95ced22df80e0414a42be5ca506cdcd43a3e9a92)